src/js/main.js

```diff
@@ -0,0 +1,15 @@
+let burger = document.querySelector('.intro_menu');
+let burger_menu = document.querySelector('.navigation')
+let intro_info = document.querySelector('.intro_info');
+let soacial_links = document.querySelector('.social_links');
+let circle = document.querySelector('.circle');
```

Comment:
Use `const` instead of `let`
Use more specific queries
Use camelcase

Reply:
I`ve some question about "specific queries"